web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -0,0 +1,112 @@
+# OB/OBC Lifecycle Management Design
```

Comment:
@shirimordechay 

Please add a link to this design in the openshift-origin-design/readme.md file.  Thanks.